content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -49,21 +52,25 @@ meta:
 - `deps`: List of <abbr>dependency</abbr> entries for this stage, only present
   when `dvc import` and `dvc import-url` are used. Typically there is only one
   (but several can be added manually).
+- `wdir` (optional): Working directory for the stage command to run in (relative
```

Comment:
not sure we need to specify `optional` here

Reply:
I think we do because wdir is not added by any command to this file anymore. But you can add it manually (like meta).

Reply:
Although... Unlike meta, wdir has an effect on the way DVC interprets the file. So not sure. Maybe "optional" just isn't the best word for either field.

Reply:
yes, let's just omit it for simplicity. We can mention explicity that this field can be added manually.